features/mbedtls/targets/TARGET_CRYPTOCELL310/ecdsa_alt.c

```diff
@@ -175,11 +176,12 @@ int mbedtls_ecdsa_verify( mbedtls_ecp_group *grp,
     uint32_t signature_size = ( ( grp->nbits + 7 ) / 8 ) * 2;
     const CRYS_ECPKI_Domain_t*  pDomain =  CRYS_ECPKI_GetEcDomain ( convert_mbedtls_grp_id_to_crys_domain_id( grp->id ) );
 
+#if defined(MBEDTLS_HAVE_INT64)
```

Comment:
Please note that `MBEDTLS_HAVE_INT64` does not mean that size_t is 64 bits. It is simply the result of some directive to check whether the underlying platform supports uint64_t. We had issues because of this in the past: https://github.com/ARMmbed/mbedtls/pull/1043